### PR TITLE
Activate pushing of images

### PR DESCRIPTION
**Pull request by froesef**
Signed-off-by: Florian Froese <ffr@zurich.ibm.com>

### CODE REVIEW

**Comment by roee88** [comment]
How come `Build / Push images (push)` is actually running in this PR?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -134,16 +134,20 @@ jobs:
...
-#        DOCKER_PUBLIC_TAGNAME:  ${{ steps.version.outputs.version }}
-#      run: make docker-retag-and-push-public && make helm-push-public
-#    - name: Publish latest image tag on push tag event
-#      if: github.event_name != 'pull_request' && steps.version.outputs.push_tag_event == 'true'
-#      env:
-#        DOCKER_PUBLIC_TAGNAME: 'latest'
-#      run: make docker-retag-and-push-public && make helm-push-public
+    - name: Publish images
+      if: ${{ github.event_name != 'pull_request' }}
+      env:
```
Comment (revit13) [inline .github/workflows/build.yml:145]: doesn't the if in line 114 cover this condition (`github.event_name != 'pull_request'`)?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -134,16 +134,20 @@ jobs:
...
+        DOCKER_PUBLIC_TAGNAME:  ${{ steps.version.outputs.version }}
+      run: make docker-retag-and-push-public && make helm-push-public
+    - name: Publish latest image tag on push tag event
+      if: github.event_name != 'pull_request' && steps.version.outputs.push_tag_event == 'true'
```
Comment (revit13) [inline .github/workflows/build.yml]: same question as above...

**Comment by froesef** [comment]
> How come `Build / Push images (push)` is actually running in this PR?

I pushed the branch to origin and not to my fork. So it ran the job from the commit (push) action.
That way I could test if a push would actually work. (And currently it does not because we need to remove some artifacts first)

**Pull request merged**